### PR TITLE
Add an ErgoDox bépo layout.

### PR DESCRIPTION
**Pull request by gid0**
This layout is my personal approach on making an ErgoDox layout suitable for bépo (a french layout based on Dvorak rules). It also features a compatibility layer to have a bépo-like layout on qwerty systems.

### CODE REVIEW

File: keyboard/ergodox_ez/keymaps/bepo/keymap.c [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+KC_NO,		KC_TRNS,	KC_NO)
+};
+
+const uint16_t PROGMEM fn_actions[] = {
+};
+
+const macro_t *action_get_macro(keyrecord_t *record, uint8_t id, uint8_t opt)
+{
+  switch(id) {
+    case KP_00:
```
Comment (ezuk) [inline keyboard/ergodox_ez/keymaps/bepo/keymap.c]: What's this do? :) Might be worth a comment here or maybe in the readme.

File: keyboard/ergodox_ez/keymaps/bepo/keymap.c [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+      if (record->event.pressed) {
+        return MACRO( T(KP_0), D(KP_0), END );
+      } else {
+        return MACRO( U(KP_0), END );
+      }
+      break;
+    case CA_Fx:
```
Comment (ezuk) [inline keyboard/ergodox_ez/keymaps/bepo/keymap.c]: And this guy?

**Comment by ezuk** [comment]
Love this! Very detailed commits, too. My only request/suggestion: Would you be willing to add a Readme with some of the features/what makes this keymap awesome? I am working on a Community Contributions section on the official website and would like to feature your keymap.

**Comment by gid0** [comment]
Will do this. Thanks for your comments !

**Comment by ezuk** [comment]
@gid0 just a quick followup, would love to merge this :)

**Comment by gid0** [comment]
I'm on it, working on an english / french README.

**Comment by ezuk** [comment]
Hi @gid0 - actually a single language would be fine too! :) Would love to merge this in this week if possible.

**Pull request merged**